taca/storage/cli.py

```diff
@@ -5,7 +5,8 @@
 
 
 @click.group()
-@click.option('-d', '--days', type=click.INT, help="Days to consider as thershold")
+@click.option('-d', '--days', type=click.INT, help="Days to consider as thershold, this should not be used \
+                                                    if 'cleanup' sub-command is called with '--hours'")
```

Comment:
I put `--hours` for `cleanup` as it will (meant to) be used only there, where `--days` are used in both sub-commands `archive` and `cleanup`.

But I can put it here, as it would look neat :+1: